src/windows-nano/1803/helix/amd64/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM mcr.microsoft.com/powershell:6.2.0-nanoserver-1803
+
+ENTRYPOINT C:\Windows\System32\cmd.exe
+
+WORKDIR C:\\Setup
+
+# Get remote files
+ADD "https://www.nuget.org/api/v2/package/python/3.7.3" python-3.7.3-nuget-amd64.zip
```

Comment:
This is not following the Dockerfile best practices because even though you delete the downloaded zip in a later layer, you still carry it in your image within this layer.  You should use something similar to the following pattern.

```
RUN Invoke-WebRequest -OutFile dotnet.zip https://dotnetcli.blob.core.windows.net/dotnet/Runtime/$Env:DOTNET_VERSION/dotnet-runtime-$Env:DOTNET_VERSION-win-x64.zip; `
    Expand-Archive dotnet.zip -DestinationPath dotnet; `
    Remove-Item -Force dotnet.zip
```

Reply:
done